include/cvc5/cvc5_skolem_id.h

```diff
@@ -171,6 +171,40 @@ enum ENUM(SkolemId)
    * - Sort: ``Real``
    */
   EVALUE(TRANSCENDENTAL_SINE_PHASE_SHIFT),
+  /**
+   * Used to reason about virtual term substitution. This term represents
+   * an infinitessemal.
+   *
+   * - Number of skolem indices: ``0``
+   * - Sort: ``Real``
+   */
+  EVALUE(ARITH_VTS_DELTA),
+  /**
+   * Used to reason about virtual term substitution. This term represents
+   * an infinitessemal.
+   *
+   * - Number of skolem indices: ``0``
+   * - Sort: ``Real``
+   */
+  EVALUE(ARITH_VTS_DELTA_FREE),
```

Comment:
How is this different from `ARITH_VTS_DELTA`?
```suggestion
  /**
   * Used to reason about virtual term substitution. This term represents
   * an infinitesimal.
   *
   * - Number of skolem indices: ``0``
   * - Sort: ``Real``
   */
  EVALUE(ARITH_VTS_DELTA_FREE),
```